config/jobs/kubernetes-sigs/ibm-vpc-block-csi-driver/OWNERS

```diff
@@ -0,0 +1,7 @@
+
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - arahamad
+reviewers:
+  - arahamad
```

Comment:
Can you please add one more person so that @arahamad isn't a single point of failure here?

Reply:
@spiffxp Thanks for reviewing the PR.  Org membership request is pending https://github.com/kubernetes/org/pull/2884

I will update the OWNERS once it is approved.
Thanks

Reply:
https://github.com/kubernetes/org/pull/2891 should be merging shortly, they'll have to accept the invite once that gets sent

Reply:
@spiffxp Could you please take a look now.